src/index.css

```diff
@@ -4,6 +4,9 @@ body {
     'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #462d37;
+  max-width: 100% !important;
+  overflow-x: hidden !important;
```

Comment:
These two important rules don't seem to be doing much. As a general rule of thumb you want to avoid using `!important` entirely. Ideally use specificity to handle style overrides.